Makefile

```diff
@@ -151,23 +151,23 @@ test: manifests generate fmt vet envtest ## Run tests.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./cmd/kubestellar-operator/main.go $(ARGS)
 
-.PHONY: ko-build
-ko-build: test ## Build docker image with ko
-	KO_DOCKER_REPO=ko.local ko build --push=false -B ./cmd/${CMD_NAME} -t ${IMAGE_TAG} --platform linux/amd64,linux/arm64
+.PHONY: ko-build-local
+ko-build-local: test ## Build local container image with ko
+	$(shell (docker version | { ! grep -qi podman; } ) || echo "DOCKER_HOST=unix://$$HOME/.local/share/containers/podman/machine/qemu/podman.sock ") KO_DOCKER_REPO=ko.local ko build -B ./cmd/${CMD_NAME} -t ${IMAGE_TAG}
```

Comment:
@MikeSpreitzer this only works for podman ... what about users using docker engine on linux or rancher desktop on Mac ?

Reply:
I will try removing the DOCKER_HOST setting from here and making it in my shell.